--- app.py.1	2013-07-12 21:30:18.000000000 +0200
+++ app.py	2013-07-11 23:45:29.334104398 +0200
@@ -4,6 +4,7 @@
 from . import TclError
 from .tclobj import TclObject, FromObj, AsObj, TypeCache
 
+from os.path import dirname, join
 import sys
 
 def varname_converter(input):
@@ -13,6 +14,12 @@
 
 
 def Tcl_AppInit(app):
+    lib_path = join(dirname(dirname(dirname(__file__))), 'lib')
+    tcl_path = join(lib_path, 'tcl')
+    tk_path = join(lib_path, 'tk')
+    tklib.Tcl_Eval(app.interp, 'set tcl_library "{0}"'.format(tcl_path))
+    tklib.Tcl_Eval(app.interp, 'set tk_library "{0}"'.format(tk_path))
+
     if tklib.Tcl_Init(app.interp) == tklib.TCL_ERROR:
         app.raiseTclError()
     skip_tk_init = tklib.Tcl_GetVar(
