--- rpython/translator/platform/linux_orig.py	2015-01-02 01:00:05.354058668 +0100
+++ rpython/translator/platform/linux.py	2015-01-02 01:00:43.276964050 +0100
@@ -26,14 +26,12 @@
 
     def _include_dirs_for_libffi(self):
         return self._pkg_config("libffi", "--cflags-only-I",
-                                ['/usr/include/libffi'],
-                                check_result_dir=True)
+                                ['/usr/include/libffi'])
 
     def _library_dirs_for_libffi(self):
         return self._pkg_config("libffi", "--libs-only-L",
-                                ['/usr/lib/libffi'],
-                                check_result_dir=True)
-
+                                ['/usr/lib/libffi'])
+   
 
 class Linux(BaseLinux):
     if platform.machine().startswith('arm'):
