--- lib-python/2.7/distutils/sysconfig_pypy.orig.py	2017-04-04 14:41:03.829620959 +0000
+++ lib-python/2.7/distutils/sysconfig_pypy.py	2017-04-04 14:42:24.221028972 +0000
@@ -206,6 +206,9 @@
         else:
             archiver = ar + ' ' + ar_flags
 
+        libdir = os.path.join(sys.prefix, 'lib')
+        ldshared = ldshared + ' -L' + libdir + ' -Wl,-rpath,' + libdir
+
         cc_cmd = cc + ' ' + cflags
         compiler.set_executables(
             preprocessor=cpp,

